src/components/Calculator.js

```diff
@@ -0,0 +1,32 @@
+const Calculator = () => {
+  const arr = [
+    { btn: 'AC' },
+    { btn: '+/-' },
+    { btn: '%' },
+    { btn: '/' },
+    { btn: 7 },
+    { btn: 8 },
+    { btn: 9 },
+    { btn: '*' },
+    { btn: 4 },
+    { btn: 5 },
+    { btn: 6 },
+    { btn: '-' },
+    { btn: 1 },
+    { btn: 2 },
+    { btn: 3 },
+    { btn: '+' },
+    { btn: 0 },
+    { btn: '.' },
+    { btn: '=' },
+  ];
+  return (
+    <div className="container">
+      <input type="text" readOnly placeholder="0" />
+      <div className="keys">
+        {arr.map((element, index) => (<button id={index} key={element.btn} type="button">{element.btn}</button>))}
+      </div>
+    </div>
+  );
+};
```

Comment:
- [ ] Please create at least one child component as requested in the project requirements, I would suggest placing the buttons into a separate component.

![image](https://user-images.githubusercontent.com/83803414/226700463-3307369c-70dd-4f97-9b75-4e5c5773e826.png)